pkg/scheduler/core/generic_scheduler.go

```diff
@@ -718,16 +718,15 @@ func (g *genericScheduler) prioritizeNodes(
 	meta interface{},
 	nodes []*v1.Node,
 ) (framework.NodeScoreList, error) {
-	// If no priority configs are provided, then the EqualPriority function is applied
+	// If no priority configs are provided, then all nodes will have a score of one.
 	// This is required to generate the priority list in the required format
 	if len(g.prioritizers) == 0 && len(g.extenders) == 0 && !g.framework.HasScorePlugins() {
 		result := make(framework.NodeScoreList, 0, len(nodes))
 		for i := range nodes {
```

Comment:
nit: `for _, node := range nodes`

Do we even have to do this? What if we just return an empty list and then select a random node in `selectHost`? 
https://github.com/kubernetes/kubernetes/blob/c33f217a5d62d062c069cd702bb50ff8d120a168/pkg/scheduler/core/generic_scheduler.go#L288-L293

Reply:
never mind, it's not important at all. We can probably do some cleanup once we completely migrate to the framework.